qt/scientific_interfaces/Indirect/IndirectSqw.cpp

```diff
@@ -17,6 +17,10 @@ using MantidQt::API::BatchAlgorithmRunner;
 
 namespace {
 
+bool doesExistInADS(std::string const &workspaceName) {
```

Comment:
I don't see the point in creating a one-line wrapper around a one-line call.

Reply:
The only reason is I find it to be more readable. However I am happy to change this if you'd like?